src/main/java/core/basesyntax/RunApplication.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class RunApplication {
+    private static FigureSupplier figureSuppluer = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSuppluer.getRandomFigure();
+        }
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/191811810-03e5e1ba-b1f6-457b-8332-9feeb7b288bf.png)
lets also add public method to FigureSupplier getDefaultFigure() and invoke it here too